src/core-sdk-tasks/CalculateTemplateVersions.cs

```diff
@@ -116,15 +110,17 @@ private static NuGetVersion GetBaseMajorMinorPatch(NuGetVersion aspNetCoreTempla
             // more in the above bug's detail.
             // There is no non-deterministic existing ComponentId under Major version 5.
             // so only apply the patch bump when below 5
-
-            int basePatch =
-                aspNetCoreTemplate.Major < 5
-                ? aspNetCoreTemplate.Patch + _patchVersionResetOffset
-                : aspNetCoreTemplate.Patch;
-
-            var baseMajorMinorPatch = new NuGetVersion(aspNetCoreTemplate.Major, aspNetCoreTemplate.Minor,
-                basePatch);
-            return baseMajorMinorPatch;
+            int basePatch = aspNetCoreTemplate.Major < 5 ?
+                aspNetCoreTemplate.Patch + PatchVersionResetOffset :
+                aspNetCoreTemplate.Patch;
+            return new NuGetVersion(aspNetCoreTemplate.Major, aspNetCoreTemplate.Minor, basePatch);
         }
     }
+
+    public class BundledTemplate
+    {
+        public string InstallPath { get; set; }
+        public string MajorMinorVersion { get; set; }
+        public string MajorMinorPatchVersion { get; set; }
+    }
```

Comment:
I originally tried to use a Record here but this is a multi-target build (it also builds against `net472`) and doesn't have records available. Using this instead of a ValueTuple made the code above a bit easier to read and the tests easier to read. No functional changes.